edx_django_utils/monitoring/internal/backends.py

```diff
@@ -70,6 +70,12 @@ def tag_root_span_with_error(self, exception):
         be implemented for OTEL.
         """
 
+    @abstractmethod
+    def set_current_span_name(self, name, group=None, priority=None):
```

Comment:
For NR, the "transaction" most closely maps to what Datadog calls the local root span. I don't think we want to modify the current span in either case.

```suggestion
    def set_local_root_span_name(self, name, group=None, priority=None):
```

Reply:
renamed it.